samples/tutorial/Tutorial2.md

```diff
@@ -212,18 +217,18 @@ Now that there are two different screens, we can make our first workflow showing
 
 Create a new workflow called `Root` with the templates.
 
-We'll start with the `RootWorkflow` returning a rendering only showing the `WelcomeScreen` via the `WelcomeWorkflow`. Update the `Rendering` type and `render` to have the `RootWorkflow` defer to a child:
+We'll start with the `RootWorkflow` returning a rendering only showing the `WelcomeScreen` via the `WelcomeWorkflow`. Update the `Rendering` type to `Any` and `render()` to return `Any`. This will allow us to eventually render multiple screens each of a diferent type. Also update `render()` to have the `RootWorkflow` defer to a child workflow:
```

Comment:
```suggestion
We'll start with the `RootWorkflow` returning a rendering only showing the `WelcomeScreen` via the `WelcomeWorkflow`. Update the `Rendering` type to `Any` and `render()` to return `Any`. This will allow us to eventually render multiple screens each of a different type. Also update `render()` to have the `RootWorkflow` defer to a child workflow:
```